src/components/Dropdown/components/singleValue/singleValue.jsx

```diff
@@ -5,11 +5,10 @@ import { components } from "react-select";
 import { ChildrenContent } from "../ChildrenContent/ChildrenContent";
 import styles from "./singleValue.module.scss";
 
-const SingleValue = props => {
-  const { Renderer, data, children, readOnly, singleValueWrapperClassName } = props;
-
+const SingleValue = ({ Renderer, data, children, readOnly, singleValueWrapperClassName, ...props }) => {
+  const rendererProps = { children, readOnly, data, ...props };
```

Comment:
Can you please elaborate why is this change needed?

Reply:
So it would be possible to render a custom `SingleValue` using `valueRenderer`, before it wasn't getting all the props